README.md

```diff
@@ -162,8 +162,8 @@ def call_model(request):
     instance_prompt = request_body.get("instance_prompt")
     run_id = request_body.get("run_id")
     run = FinetuningRun(run_id)
-    model = StableDiffusionPipeline(model_id=run.deployed_model.id)
-    image, url = model(instance_prompt)
+    model = run.deployed_model
+    _, url = model(instance_prompt)
```

Comment:
let's keep the `image` even though it's unused. it's informational for the reader.